test/verifiedTests/base/testOutputNetworkCytoscape/testOutputNetworkCytoscape.m

```diff
@@ -19,9 +19,18 @@
 
 load([CBTDIR, filesep, 'test' filesep 'models' filesep 'ecoli_core_model.mat'], 'model');
 
+%additional tests to check all options
+outputNetworkCytoscape(model, 'data');
+outputNetworkCytoscape(model, 'data', [], [], [], [], 0); %up to 8
+outputNetworkCytoscape(model, 'data', model.rxns, {'1'}, model.mets, {'1'}, 0);
+outputNetworkCytoscape(model, 'data', model.rxns, ['1'], model.mets, ['1'], 0);
+outputNetworkCytoscape(model, 'data', model.rxns, ['1', '2'], model.mets, [], 0);
+outputNetworkCytoscape(model, 'data', model.rxns, {'1', '2'}, model.mets, {'1', '2'}, 0);
+outputNetworkCytoscape(model, 'data', model.rxns, ['1'], model.mets, ['1', '2'], 0);
+outputNetworkCytoscape(model, 'data', model.rxns, [], model.mets, ['1'], 0);
```

Comment:
I added the tests for outputs with a loop since there are only two possibilities - either it is empty or is equal to model.mets